content/tr/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Cloud Native Sözlüğü"
+status: Completed
+---
+
+**Cloud Native Sözlüğü**
+
+Cloud Native Sözlüğü, karmaşıklığı ile ünlü olan cloud native alanının yalnızca teknoloji uzmanları için değil, herkes için anlaşılabilir olmasını hedeflemektedir. Basitliğe odaklanan sözlük, CNCF Business Value Subcommittee (BVS) tarafından yönetilen bir projedir.
+
```

Comment:
```suggestion
Cloud Native Sözlüğü, karmaşıklığı ile ünlü olan cloud native alanının yalnızca teknoloji uzmanları için değil, herkes için anlaşılabilir olmasını hedeflemektedir. Basitliğe odaklanan sözlük, CNCF Business Value Subcommittee (BVS) tarafından yönetilen bir projedir.

<p><img class="mt-3" src="/images/homepage/kubecon.jpg" alt="Kubecon sunumu izleyen insanlar"></p>
```

We missed this image in the Google Doc (which is fine).

In general, we should compare the page with the English version (https://raw.githubusercontent.com/cncf/glossary/8df5004a00fc1a78ad2914665e691cfee22a6a47/content/en/_index.md)

Reply:
Thank you.